app/models/test_track/remote/split_registry.rb

```diff
@@ -3,7 +3,7 @@ class TestTrack::Remote::SplitRegistry
 
   CACHE_KEY = 'test_track_split_registry'.freeze
 
-  collection_path '/api/v2/split_registry'
+  collection_path '/api/v3/builds/:build_timestamp/split_registry'
```

Comment:
Is this used in any way? I think every usecase should go through the instance method below but we should confirm.

Reply:
both https://github.com/reggieag/test_track_rails_client/blob/rg/add-timestamp/app/models/test_track/remote/visitor_detail.rb and https://github.com/reggieag/test_track_rails_client/blob/rg/add-timestamp/app/models/test_track/remote/visitor.rb follow this pattern but the rest of the remote models rely only on collection_path. I wonder when we wanted to start potentially ignoring collection_path. is it some sort of HER framework requirement or a documentation like attribute? It does seem superfluous if we're just using the call on line 18

Reply:
Her doesn't actually support singular routes I think, so we duplicate when we have to IIRC, just so it's clear what the route is we're talking about.